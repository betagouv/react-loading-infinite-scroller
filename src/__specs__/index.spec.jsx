import React from 'react'
import { shallow } from 'enzyme'

import { LoadingInfiniteScroll } from '../index'

describe('test LoadingInfiniteScroll', () => {
  describe('snapshot', () => {
    it('should match snapshot', () => {
      // given
      const props = {
        handlePageChange: jest.fn(),
        handlePageReset: jest.fn(),
      }

      // when
      const wrapper = shallow(
        <LoadingInfiniteScroll {...props}>
          <div />
        </LoadingInfiniteScroll>
      )

      // then
      expect(wrapper).toBeDefined()
      expect(wrapper).toMatchSnapshot()
    })
  })
})
