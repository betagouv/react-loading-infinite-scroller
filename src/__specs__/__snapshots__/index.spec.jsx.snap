// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test LoadingInfiniteScroll snapshot should match snapshot 1`] = `ShallowWrapper {}`;
